pandas/core/arrays/base.py

```diff
@@ -820,19 +820,30 @@ def take(self, indices, allow_fill=False, fill_value=None):
         # pandas.api.extensions.take
         raise AbstractMethodError(self)
 
-    def copy(self, deep: bool = False) -> ABCExtensionArray:
+    def copy(self) -> ABCExtensionArray:
```

Comment:
We need to deprecate `deep` right? Or is in not in 0.24?

Reply:
Oh, just saw https://github.com/pandas-dev/pandas/pull/27083#pullrequestreview-255410112.

Slight preference for deprecation, but don't have strong thoughts here.